### ISSUE TITLE
Wurst crash

### ISSUE THREAD

**Issue by liledusty** [Wurst-Imperium/Wurst-Client-Bugs#62]
#### Description of the Bug
```
Description: Initializing game

java.lang.OutOfMemoryError: Java heap space
	at java.awt.image.DataBufferInt.<init>(Unknown Source)
	at java.awt.image.Raster.createPackedRaster(Unknown Source)
	at java.awt.image.DirectColorModel.createCompatibleWritableRaster(Unknown Source)
	at java.awt.image.BufferedImage.<init>(Unknown Source)
	at net.minecraft.client.renderer.texture.TextureUtil.func_177055_a(TextureUtil.java:369)
	at net.minecraft.client.renderer.texture.TextureMap.loadTextureAtlas(TextureMap.java:278)
	at net.minecraft.client.renderer.texture.TextureMap.func_174943_a(TextureMap.java:83)
	at net.minecraft.client.resources.model.ModelBakery.func_177572_j(ModelBakery.java:666)
	at net.minecraft.client.resources.model.ModelBakery.setupModelRegistry(ModelBakery.java:87)
	at net.minecraft.client.resources.model.ModelManager.onResourceManagerReload(ModelManager.java:26)
	at net.minecraft.client.resources.SimpleReloadableResourceManager.registerReloadListener(SimpleReloadableResourceManager.java:121)
	at net.minecraft.client.Minecraft.startGame(Minecraft.java:510)
	at net.minecraft.client.Minecraft.run(Minecraft.java:359)
	at net.minecraft.client.main.Main.main(Main.java:129)
```

#### How to Reproduce the Bug
```
- Head --
Stacktrace:
	at java.awt.image.DataBufferInt.<init>(Unknown Source)
	at java.awt.image.Raster.createPackedRaster(Unknown Source)
	at java.awt.image.DirectColorModel.createCompatibleWritableRaster(Unknown Source)
	at java.awt.image.BufferedImage.<init>(Unknown Source)
	at net.minecraft.client.renderer.texture.TextureUtil.func_177055_a(TextureUtil.java:369)
	at net.minecraft.client.renderer.texture.TextureMap.loadTextureAtlas(TextureMap.java:278)
	at net.minecraft.client.renderer.texture.TextureMap.func_174943_a(TextureMap.java:83)
	at net.minecraft.client.resources.model.ModelBakery.func_177572_j(ModelBakery.java:666)
	at net.minecraft.client.resources.model.ModelBakery.setupModelRegistry(ModelBakery.java:87)
	at net.minecraft.client.resources.model.ModelManager.onResourceManagerReload(ModelManager.java:26)
	at net.minecraft.client.resources.SimpleReloadableResourceManager.registerReloadListener(SimpleReloadableResourceManager.java:121)
	at net.minecraft.client.Minecraft.startGame(Minecraft.java:510)

-- Initialization --
Details:
Stacktrace:
	at net.minecraft.client.Minecraft.run(Minecraft.java:359)
	at net.minecraft.client.main.Main.main(Main.java:129)

-- System Details --
Details:
	Minecraft Version: 1.8
	Wurst Client Version: ~~ERROR~~ NullPointerException: null
	Operating System: Windows 7 (amd64) version 6.1
	Java Version: 1.8.0_91, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 218555488 bytes (208 MB) / 1060372480 bytes (1011 MB) up to 1060372480 bytes (1011 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	Launched Version: Wurst
	LWJGL: 2.9.1
	OpenGL: GeForce GTX 750 Ti/PCIe/SSE2 GL version 4.4.0 NVIDIA 344.75, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: No
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	Resource Packs: [Gerwazus TxT [1.7 1.8] v2.0]
	Current Language: English (UK)
	Profiler Position: N/A (disabled)
```

#### Other Information
- Wurst Client version: in all versions :v
- Operating system: Windows 7

---

I have read and understood the [Bug Tracker Rules](https://github.com/Wurst-Imperium/Wurst-Client-Bugs/blob/master/.github/CONTRIBUTING.md) before posting this bug report and filled out the template above with the correct information.

**Comment by mcccs** [comment]
What did you do before it crashed?

Please insert three ` icons before and after the log. This is what it has to be:
```java
Description: Initializing game

java.lang.OutOfMemoryError: Java heap space
at java.awt.image.DataBufferInt.(Unknown Source)
at java.awt.image.Raster.createPackedRaster(Unknown Source)
at java.awt.image.DirectColorModel.createCompatibleWritableRaster(Unknown Source)
at java.awt.image.BufferedImage.(Unknown Source)
at net.minecraft.client.renderer.texture.TextureUtil.func_177055_a(TextureUtil.java:369)
at net.minecraft.client.renderer.texture.TextureMap.loadTextureAtlas(TextureMap.java:278)
at net.minecraft.client.renderer.texture.TextureMap.func_174943_a(TextureMap.java:83)
at net.minecraft.client.resources.model.ModelBakery.func_177572_j(ModelBakery.java:666)
at net.minecraft.client.resources.model.ModelBakery.setupModelRegistry(ModelBakery.java:87)
at net.minecraft.client.resources.model.ModelManager.onResourceManagerReload(ModelManager.java:26)
at net.minecraft.client.resources.SimpleReloadableResourceManager.registerReloadListener(SimpleReloadableResourceManager.java:121)
at net.minecraft.client.Minecraft.startGame(Minecraft.java:510)
at net.minecraft.client.Minecraft.run(Minecraft.java:359)
at net.minecraft.client.main.Main.main(Main.java:129)

How to Reproduce the Bug

Head -- Stacktrace: at java.awt.image.DataBufferInt.(Unknown Source) at java.awt.image.Raster.createPackedRaster(Unknown Source) at java.awt.image.DirectColorModel.createCompatibleWritableRaster(Unknown Source) at java.awt.image.BufferedImage.(Unknown Source) at net.minecraft.client.renderer.texture.TextureUtil.func_177055_a(TextureUtil.java:369) at net.minecraft.client.renderer.texture.TextureMap.loadTextureAtlas(TextureMap.java:278) at net.minecraft.client.renderer.texture.TextureMap.func_174943_a(TextureMap.java:83) at net.minecraft.client.resources.model.ModelBakery.func_177572_j(ModelBakery.java:666) at net.minecraft.client.resources.model.ModelBakery.setupModelRegistry(ModelBakery.java:87) at net.minecraft.client.resources.model.ModelManager.onResourceManagerReload(ModelManager.java:26) at net.minecraft.client.resources.SimpleReloadableResourceManager.registerReloadListener(SimpleReloadableResourceManager.java:121) at net.minecraft.client.Minecraft.startGame(Minecraft.java:510)
-- Initialization --
Details:
Stacktrace:
at net.minecraft.client.Minecraft.run(Minecraft.java:359)
at net.minecraft.client.main.Main.main(Main.java:129)

-- System Details --
Details:
Minecraft Version: 1.8
Wurst Client Version: ERROR NullPointerException: null
Operating System: Windows 7 (amd64) version 6.1
Java Version: 1.8.0_91, Oracle Corporation
Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
Memory: 218555488 bytes (208 MB) / 1060372480 bytes (1011 MB) up to 1060372480 bytes (1011 MB)
JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
Launched Version: Wurst
LWJGL: 2.9.1
OpenGL: GeForce GTX 750 Ti/PCIe/SSE2 GL version 4.4.0 NVIDIA 344.75, NVIDIA Corporation
GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

Using VBOs: No
Is Modded: Very likely; Jar signature invalidated
Type: Client (map_client.txt)
Resource Packs: [Gerwazus TxT [1.7 1.8] v2.0]
Current Language: English (UK)
Profiler Position: N/A (disabled)
```

**Comment by Alexander01998** [comment]
This is not a bug, your computer just doesn't have enough RAM to run Minecraft.
